kong.conf.default

```diff
@@ -180,31 +200,49 @@
                          # 'data-plane' mode (without configuration
                          # capabilities) pulling its configuration changes
                          # from the database.
+                         #
+                         # Example: `127.0.0.1:8444 http2 ssl`
 
 #nginx_user = nobody nobody      # Defines user and group credentials used by
                                  # worker processes. If group is omitted, a
                                  # group whose name equals that of user is
-                                 # used. Ex: [user] [group].
+                                 # used.
+                                 #
+                                 # Example: `nginx www`
 
 #nginx_worker_processes = auto   # Determines the number of worker processes
                                  # spawned by Nginx.
+                                 #
+                                 # See http://nginx.org/en/docs/ngx_core_module.html#worker_processes for detailed
```

Comment:
"for detailed" should be on the next line. Reminder that we try to _strongly_ respect the 80 chars limit in this file. Only links over that limit are exception to this rule.